public/app/features/explore/Logs.tsx

```diff
@@ -180,6 +177,13 @@ export class UnthemedLogs extends PureComponent<Props, State> {
     }
   };
 
+  handleOnChangeAutoLogsVolume = (event?: React.SyntheticEvent) => {
+    const target = event && (event.target as HTMLInputElement);
```

Comment:
I just used the same convention as for other options to keep it consistent. I can change it, I'll update other options as well.

- [ ] simplify event handlers to avoid casting

Reply:
@gabor Though I wonder why the current signature is `event?: React.SyntheticEvent`. Could event really be `undefined`? 🤔

Reply:
@ifrost , i did the "scientific" 😄   thing, and modified that line to be:
```
                <InlineSwitch
                  value={autoLoadLogsVolume}
                  onChange={(thing) => {
                    // FIXME
                  }}
                  transparent
                />
```

and then mouse-overed on the `thing` part in vs-code, and it said it's type is `React.FormEvent<HTMLInputElement>` ... so i do not think it can be nullish.

Reply:
and i do not know why it is `event?` .. i tried and deleted the `?` from every `event?` and vscode did not complain...

Reply:
Done in 0f1b4908dacd9c2687ac3ab99ba16190fa335e50